src/frost.rs

```diff
@@ -389,14 +389,18 @@ pub struct SigningPackage {
     pub message: &'static [u8],
 }
 
+/// A representation of a single signature used in FROST structures and messages.
+#[derive(Clone, Copy, Default)]
+pub struct FrostSignature(Scalar);
```

Comment:
```suggestion
pub struct SignatureShare(Scalar);
```

Reply:
Changed them all to `SignatureResponse` as i think that was the intention. https://github.com/ZcashFoundation/redjubjub/pull/107/commits/f12511092c9bb73d8fbe8d55157479e6805d8105